.changeset/config.json

```
@@ -2,10 +2,6 @@
   "$schema": "https://unpkg.com/@changesets/config@2.1.1/schema.json",
   "changelog": "@changesets/cli/changelog",
   "commit": false,
```

Can we keep the `@talismn/balances` & `@talismn/balances-react` versions in sync?

```suggestion
  "commit": false,
  "fixed": [
    ["@talismn/balances", "@talismn/balances-*"]
  ],
```

---

sorry my bad, thought it was a leftover from the old per-module balance libraries